core/dropdowndiv.js

```diff
@@ -309,6 +311,8 @@ Blockly.DropDownDiv.show = function(owner, primaryX, primaryY,
  * @private
  */
 Blockly.DropDownDiv.getBoundsInfo_ = function() {
+  // TODO: Account for toolboxes.
```

Comment:
Is this new? Where we accounting for toolboxes earlier?

Reply:
This is new, see #2744 for details.